Fixtures/ProducerManagerStub.php

```diff
@@ -0,0 +1,54 @@
+<?php
+declare(strict_types = 1);
+
+namespace M6Web\Bundle\KafkaBundle\Fixtures;
+
+use M6Web\Bundle\KafkaBundle\AbstractManager;
+use M6Web\Bundle\KafkaBundle\Producer\ProducerManager;
+
+/**
+ * Class ProducerManagerStub
+ * @package M6Web\Bundle\KafkaBundle\Fixtures
+ *
+ * A stub class to imitate ProducerManager class
+ */
+class ProducerManagerStub extends ProducerManager
+{
+    /**
+     * @param \RdKafka $entity
+     * @return AbstractManager
+     */
+    public function setEntity(\RdKafka $entity): AbstractManager
+    {
+        return $this;
+    }
+
+    /**
+     * @param string $brokers
+     * @return AbstractManager
+     */
+    public function addBrokers(string $brokers): AbstractManager
+    {
+        return $this;
+    }
+
+    /**
+     * @param int $logLevel
+     * @return AbstractManager
+     */
+    public function setLogLevel(int $logLevel): AbstractManager
+    {
+        return $this;
+    }
+
+    /**
+     * @param string             $name
+     * @param \RdKafka\TopicConf $rdKafkaTopicConf
+     * @param array|null         $confToSet
+     * @return $this
```

Comment:
quand les IDE et php7 seront assez matures, on pourra sûrement se passer des `@param` et `@return`, pour le moment, l'un nous met des warnings quand on oublie la phpdoc, l'autre nous empêche d'indiquer qu'on retourne l'instance courante ($this), donc IMHO y a pas de solution miracle